src/components/Frame/tests/Frame.test.tsx

```diff
@@ -59,16 +54,10 @@ describe('<Frame />', () => {
   });
 
   it('renders a TrapFocus with a `trapping` prop set to false prop around the navigation on small screens and showMobileNavigation is false', () => {
-    Object.defineProperty(window, 'innerWidth', {
-      configurable: true,
-      writable: true,
-      value: 500,
-    });
-
     const navigation = <div />;
-    const frame = mountWithAppProvider(<Frame navigation={navigation} />).find(
-      Frame,
-    );
+    const frame = mountWithAppProvider(<Frame navigation={navigation} />, {
+      mediaQuery: {isNavigationCollapsed: false},
```

Comment:
This isn't really needed since it'll default to false, but since we're testing small screens I left it in to be declarative.